File: Apuntes/Tercera Parcial/calculadora.erl
-module(calculadora). 
-export([suma/2, resta/2, multiplicacion/2, division/2]).

suma(N1, N2)-> N1+N2.
resta(N1, N2)-> N1-N2.
multiplicacion(N1, N2)-> N1*N2.
division(N1, N2)-> N1/N2.
%variables destructivas

File: Apuntes/Tercera Parcial/Calculadora.exs
defmodule Calculadora do
  def suma(n1, n2) do
    n1+n2
  end
  def resta(n1, n2) do
    n1-n2
  end
  def multiplicacion(n1, n2) do
    n1*n2
  end
  def division(_, n2) when n2==0 do
    "Division por cero"
  end
  def division(n1, n2) do
    n1/n2
  end
end
